FILES/GetTattooShopDlcItemData.md

```diff
@@ -9,51 +9,37 @@ aliases: ["0xFF56381874F82086","_GET_TATTOO_COLLECTION_DATA"]
 BOOL GET_TATTOO_SHOP_DLC_ITEM_DATA(int characterType, int decorationIndex, Any* outComponent);
 ```
 
-Character types:
-```
-0 = Michael,
-1 = Franklin,
-2 = Trevor,
-3 = MPMale,
-4 = MPFemale
-```
+Returns data that adheres to the tattoo shop item data that is used in shop_tattoo.meta
 
-```csharp
-enum TattooZoneData
-{
-    ZONE_TORSO = 0,
-    ZONE_HEAD = 1,
-    ZONE_LEFT_ARM = 2,
-    ZONE_RIGHT_ARM = 3,
-    ZONE_LEFT_LEG = 4,
-    ZONE_RIGHT_LEG = 5,
-    ZONE_UNKNOWN = 6,
-    ZONE_NONE = 7,
-};
+Character types:
+| Value |   Character   |
+| :---: | :-----------: |
+| `0`   | Michael       |
+| `1`   | Franklin      |
+| `2`   | Trevor        |
+| `3`   | MPMale        |
+| `4`   | MPFemale      |
 
-struct outComponent
-{
-    // these vars are suffixed with 4 bytes of padding each.
-    uint unk;
-    int unk2;
-    uint tattooCollectionHash;
-    uint tattooNameHash;
-    int unk3;
-    TattooZoneData zoneId;
-    uint unk4;
-    uint unk5;
-    // maybe more, not sure exactly, decompiled scripts are very vague around this part.
-}
-```
+Returned struct properties:
+|      Type     |     Value     |                                Information                             |
+| :-----------: | :------------:| :-------------------------------------------------------------------:  |
+| `uint`        |   LockHash    |  Lock hash, used with [`IS_CONTENT_ITEM_LOCKED`](#_0xD4D7B033C3AA243C) |
+| `int`         |   Index       |  Unique ID of this slot. Can also be 0.                                |
+| `uint`        |   Collection  |  Collection of the tattoo.                                             |
+| `uint`        |   Preset      |  Preset of this tattoo.                                                |
+| `int`         |   Cost        |  Cost of this tattoo in shops.                                         |
+| `int`         |   eFacing     |  Facing (Secondary placement value) of this tattoo.                    |
+| `uint`        |   UpdateGroup |  Location of this tattoo.                                              |
+| `const char*` | NameTextLabel |   This tattoo's name as a text label.                                  |
 
 
 ## Parameters
 * **characterType**: Character types 0 = Michael, 1 = Franklin, 2 = Trevor, 3 = MPMale, 4 = MPFemale.
```

Comment:
No need to have the character types information since it's stored on the table. You can just set `Character types`.